src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -448,10 +448,26 @@ public GHProject createProject(String name, String body) throws IOException {
     }
 
     /**
-     * The enum Permission.
+     * Permissions (roles) for teams and collaborators.
      */
-    public enum Permission {
-        ADMIN, MAINTAIN, PUSH, TRIAGE, PULL
+    public static class Permission {
+        // pre-canned permissions/roles
```

Comment:
@jgangemi 

Sorry, no. We can't change the existing email to a class as that is a binary breaking change. We can create a new class but the existing one has to stay. 

We can add an `UNKOWN` value to the enum and add a `getPermissionString()` method to allow people to get the raw value. 

But we're not ready to make a breaking change at this time.

Reply:
mm - right. forgot about that. all good, i'll do something else here.